suite-common/wallet-utils/src/solanaStakingUtils.ts

```
@@ -164,23 +128,23 @@ export const getSolStakingAccountTotalBalanceByStatus = (account: Account, statu
 export const getSolStakingAccountsInfo = (account: Account) => {
-    const balanceResults = Object.entries(StakeState).map(([key, status]) => {
+    const balanceResults = Object.values(StakeState).map(status => {
         const balance = getSolStakingAccountTotalBalanceByStatus(account, status);
 
-        return [key, balance];
+        return [status, balance];
     });
 
     const balances: Record<StakeStateType, string> = balanceResults.reduce(
-        (acc, [key, balance]) => ({ ...acc, [key]: balance }),
+        (acc, [status, balance]) => ({ ...acc, [status]: balance }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)